b2share/version.py

```diff
@@ -20,4 +20,4 @@
 """Version number."""
 
 
-__version__ = "2.0.0.rc3"  # FIXME: generate the version number
+__version__ = "2.0.0.rc4"  # FIXME: generate the version number
```

Comment:
@nharraud it should be probably `2.0.0rc4` (see https://www.python.org/dev/peps/pep-0440/#pre-releases).